code/src/java/pcgen/gui2/doomsdaybook/NameGenPanel.java

```diff
@@ -690,7 +687,7 @@ private void loadCatalogDD()
 			join.retainAll(sexes);
 			Collections.sort(join, new DataElementComperator());
 
-			Vector<RuleSet> catalogs = new Vector<RuleSet>();
+			List<RuleSet> catalogs = new ArrayList<>();
```

Comment:
Why should we avoid the diamond operator? It provides more readable, and less verbose code.